meta/no_language.yaml

```diff
@@ -0,0 +1,15 @@
+rules:
```

Comment:
can you rename the file empty_language.test.yaml?
@mschwager are we ready to have the special logic in semgrep to not use this .test.yaml extension as input rules but just as a test files?

Reply:
Yup, still tracking in https://github.com/returntocorp/semgrep/issues/2799. I haven't had the time to implement it, but I think we can go with the `.test.yaml` solution if someone has time to work on it. It should be reasonably quick to implement by modifying the logic here to also consider `.test.yaml` test files when the extension is `.yaml`: https://github.com/returntocorp/semgrep/blob/2508934/semgrep/semgrep/test.py#L282